src/client/client.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  connectWebSocket,
+  isWebSocketCloseEvent,
+  isWebSocketPingEvent,
+  isWebSocketPongEvent
+} from "../../deps.ts";
+import { BufReader } from "../../deps.ts";
+import { TextProtoReader } from "../../deps.ts";
+import { blue, red, yellow } from "../../deps.ts";
```

Comment:
if you want, you can do

```
import { BufReader, TextProtoReader, blue, red, yellow } from "../../deps.ts";
```

Reply:
same for the `import` statement above

Reply:
LOL duh... copy pasta. I'll change it in the next PR.